test/Test/Data/Array.purs

```diff
@@ -202,6 +202,14 @@ testArray = do
   log "filter should remove items that don't match a predicate"
   assert $ A.filter odd (A.range 0 10) == [1, 3, 5, 7, 9]
 
+  log "splitAt should split the array at the given number of elements"
+  assert $ A.splitAt 3 [1, 2, 3, 4, 5] == Tuple [1, 2, 3] [4, 5]
+  assert $ A.splitAt 1 [1, 2, 3] == Tuple [1] [2, 3]
+  assert $ A.splitAt 3 [1, 2, 3] == Tuple [1, 2, 3] []
+  assert $ A.splitAt 4 [1, 2, 3] == Tuple [1, 2, 3] []
+  assert $ A.splitAt 0 [1, 2, 3] == Tuple [] [1, 2, 3]
+  assert $ A.splitAt (-1) [1, 2, 3] == Tuple [] [1, 2, 3]
+
```

Comment:
There should be a test here for when the array argument is empty. I assume `splitAt n []` produces `Tuple [] []`.

Reply:
Added in 6f414d6.